great_expectations/data_context/store/inline_store_backend.py

```diff
@@ -0,0 +1,91 @@
+from typing import Any, List, Optional, Tuple
+
+from great_expectations.data_context.store.store_backend import StoreBackend
+from great_expectations.exceptions.exceptions import StoreBackendError
+from great_expectations.util import filter_properties_dict
+
+
+class InlineStoreBackend(StoreBackend):
```

Comment:
@cdkini @donaldheppner This actually underscores one of the reasons for my uncertainty about the `InlineStoreBackend` -- it appears to need `DataContext` -- do we need it?  Should we using `BaseDataContext` instead?  Better still, I feel that not having it in favor of a standard upgrade procedure might be a cleaner approach.  Thank you for consideration.

Reply:
In this case I think the DataContext reference makes sense since InlineStoreBackend is intended to store values in the DataContext (the DataContext is effectively the store, in the same way S3 is the store for the S3StoreBackend).

I'd rather the InlineStoreBackend reference DataContext and use its methods to read and manipulate values than have InlineStoreBackend bypass DataContext and manipulate the underlying values directly.